### DISCUSSION TITLE
Resizing PNG keeping "readability" of fonts.

### DISCUSSION THREAD

**Discussion by Mmisiek** [ImageMagick/ImageMagick#6929] [Help]
### ImageMagick version

7.0.10-61

### Operating system, version and so on

Windows 10 / CygWin

### Description

Hi,
I am trying to resize PNG screencapture from phone 1080x1220 by 25%. Below capture is at 50% zoom out.
<img width="823" alt="image" src="https://github.com/ImageMagick/ImageMagick/assets/4585329/7f3f1d46-99b8-4175-a8df-d928917d12f1">

I added two step approach:
```
convert screen_big.png -resize 540 -quality 100 -depth 8 PNG8:screen50.png
convert screen50.png -resize 270 -quality 100 -depth 8 PNG8:screen.png
```
but fonts on black background are blurry. Then I learned about --unsharpen option. 
So I switched to:
```
	mogrify -unsharp 0x1 -resize 270 -normalize -density 200 screen_big.png
	mogrify -unsharp 0x1 -resize 270 -normalize -density 200 screen_big.png
```
Basically, calling twice, the first time rescaled second removes blur on the font. The result is kind of better.
I am looking for advice on what would be the best use of ImageMagic. The image is basic colors so I can change depth.
<img width="436" alt="image" src="https://github.com/ImageMagick/ImageMagick/assets/4585329/896e8af2-ac0f-41f3-b6d7-f2793f7fa1f3">

**Comment by fmw42** [comment]
-density is not relevant to processing raster images, only vector ones like PDF.  

-quality 100 is not necessarily best for PNG.  See https://imagemagick.org/script/command-line-options.php#quality
